test/client/form-cache.spec.js

```diff
@@ -195,113 +195,4 @@ describe( 'Client Form Cache', () => {
-                .then( survey => {
-                    initResult = survey;
-
-                    return getFormPartsSpy.getCall( 0 ).returnValue;
-                } )
-                .then( formParts => {
-                    expect( getSpy.called ).to.equal( false );
-                    expect( setSpy.called ).to.equal( false );
-                    expect( initResult ).to.deep.equal( formParts );
-                } )
-                .then( done, done );
-        } );
-
-        it( 'bypasses the store when initializing a form in single mode', done => {
-            sandbox.stub( settings, 'type' ).get( () => 'single' );
-
-            const singleSurvey = {
-                enketoId: 'surveyA',
-            };
-
-            let initResult;
-
-            formCache.init( singleSurvey )
-                .then( survey => {
-                    initResult = survey;
-
-                    return getFormPartsSpy.getCall( 0 ).returnValue;
-                } )
-                .then( formParts => {
-                    expect( getSpy.called ).to.equal( false );
-                    expect( setSpy.called ).to.equal( false );
-                    expect( initResult ).to.deep.equal( formParts );
-                } )
-                .then( done, done );
-        } );
-
-        it( 'bypasses the store when initializing a form in edit mode', done => {
-            sandbox.stub( settings, 'type' ).get( () => 'edit' );
-
-            const editSurvey = {
-                enketoId: 'surveyA',
-                instanceId: 'instance',
-            };
-
-            let initResult;
-
-            formCache.init( editSurvey )
-                .then( survey => {
-                    initResult = survey;
-
-                    return getFormPartsSpy.getCall( 0 ).returnValue;
-                } )
-                .then( formParts => {
-                    expect( getSpy.called ).to.equal( false );
-                    expect( setSpy.called ).to.equal( false );
-                    expect( initResult ).to.deep.equal( formParts );
-                } )
-                .then( done, done );
-        } );
-
-        it( 'bypasses the store when initializing a form in view mode', done => {
-            sandbox.stub( settings, 'type' ).get( () => 'view' );
-
-            const viewSurvey = {
-                enketoId: 'surveyA',
-                instanceId: 'instance',
-            };
-
-            let initResult;
-
-            formCache.init( viewSurvey )
-                .then( survey => {
-                    initResult = survey;
-
-                    return getFormPartsSpy.getCall( 0 ).returnValue;
-                } )
-                .then( formParts => {
-                    expect( getSpy.called ).to.equal( false );
-                    expect( setSpy.called ).to.equal( false );
-                    expect( initResult ).to.deep.equal( formParts );
-                } )
-                .then( done, done );
-        } );
-    } );
```

Comment:
These tests are no longer relevant, because _none_ of the modes access the form cache anymore.